src/main/java/com/feedzai/commons/sql/abstraction/engine/impl/SqlServerEngine.java

```diff
@@ -669,7 +669,7 @@ protected void dropReferringFks(DbEntity entity) throws DatabaseEngineException
             final String sString = format(
                     "SELECT T1.TABLE_NAME, CONSTRAINT_NAME "
                             + "FROM INFORMATION_SCHEMA.CONSTRAINT_TABLE_USAGE AS T1 "
-                            + "JOIN SYS.FOREIGN_KEYS AS F "
+                            + "JOIN sys.foreign_keys AS F "
```

Comment:
This doesn't change anything when the database is using a default collation that is case insensitive.  
However, when the default collation is case insensitive, only "sys.foreign_keys" works.  
In fact, JDBC has a method to get the foreign keys and we could use that, but would need some testing...